specs/pnpm/uninstall_a_local_pnpm_package.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name: Uninstall a local NPM package
```

Comment:
nit: Could you include `pnpm` in the title for this workflow as well?

Reply:
Awesome thanks!